### DISCUSSION TITLE
Perses' logo

### DISCUSSION THREAD

**Discussion by Nexucis** [perses/perses#182] [General]
This discussion is to talk about the Perses' logo.

Ideally, it would be nice to have one once we will make a public announcement about Perses (on Twitter, or somewhere else).
So nothing urgent here. Logo will come when we are confident in what we want.

Some months ago, @AntoineThebaud tried different things based on the keywords `fire`, `sword`.

First tentative: the fire is drawing the `P` of `Perses`
![perses_logo](https://user-images.githubusercontent.com/4548045/143000060-d876811e-197d-471f-a9a3-0b0b891c6a24.png)

![perses_logo2](https://user-images.githubusercontent.com/4548045/143000068-50d6cffd-b2c1-4b52-9f2f-5c6607374679.png)

Another tentative, but without the `P` of `Perses`
![logo_perses3](https://user-images.githubusercontent.com/4548045/143000148-f7cb90be-099e-4b37-95f4-15189dddd979.png)

Note that it's just some ideas I wanted to share with you and also the opportunity to thanks @AntoineThebaud for these tentatives, it was cool to see it :). Perhaps it can give you some ideas or you like it and we should just improve it.

Of course we can try something totally different. Discussion is open :smile:

**Comment by AntoineThebaud** [comment]
Yes these are drafts I made back in april, 'didn't work on the subject since but I will try to come with a definitive version of the P-shaped logo proposal above (the main issue currently is that it's too vertical and doesn't fit well a square/round slot).

**Reply by Nexucis** [reply]
I think before you put on you too much pressure, we should wait a bit to have the opinion of others members :). Probably they will have different but interesting thoughts !

**Reply by AntoineThebaud** [reply]
Yeah I know no pressure here, I just would like to come with a clean proposal for this one. Hopefully it would help in the reflection process. Also I have other ideas in mind but I prefer to see what others would propose !

(and FYI all I'm not a graphist at all, as you may have guessed :D)

**Reply by AntoineThebaud** [reply]
![propal_examples](https://user-images.githubusercontent.com/7058693/143313738-ce1e74dc-22f8-4bc6-9cae-a18819f00191.png)

There it is :) refined shape & a bunch of examples of what we can do with it

**Reply by Nexucis** [reply]
uuh nice. Thanks for your work @AntoineThebaud.

I like the one with the red circle. But the one with the blue circle is cool too ! In a different style, I like this one too : ![logo_perses3](https://user-images.githubusercontent.com/4548045/143000148-f7cb90be-099e-4b37-95f4-15189dddd979.png)

But I'm an easy person to convince I think :D.
Probably @juliusv, @LukeTillman or @sjcobb will have different options / taste.

**Reply by roidelapluie** [reply]
Could we see examples without gradiant?

**Reply by AntoineThebaud** [reply]
Sure
![propal_examples_2](https://user-images.githubusercontent.com/7058693/143390514-9ea0bb87-b308-4c13-864b-856a3bb3cc6a.png)

**Comment by juliusv** [comment]
Sword + flame is one possible direction to go in, but there could be probably many other interesting ones. I'm still for doing a 99designs contest (or similar) to see a whole range of possibilities.

For the P-style logos, I find the flame too filigrane, which is a bit too much visual noise for a logo IMO. It also reminds me of the Grafana logo, which looks like a curled-up flame with many little spikes on it (like the flames here). But I think going for variations on that that are more simple and maybe more abstract could be interesting.

**Comment by juliusv** [comment]
Here's a totally different concept direction that is really just a concept of course (totally hastily drawn in a crooked way in Inkscape), but maybe could be an option to go into as well:

![perses-concept2](https://user-images.githubusercontent.com/538008/143447246-beb00239-c6ad-4763-ba89-bc922c441058.png)
![perses-concept](https://user-images.githubusercontent.com/538008/143447249-047bee09-8044-46e1-8ae1-f628fb47342e.png)

**Comment by juliusv** [comment]
Instead of the stripes that I drew on the "P", you could also imagine something more like these: https://i.pinimg.com/originals/70/4f/2b/704f2b9c53b01caf36eedecd6e4ade0e.png

**Reply by Nexucis** [reply]
Nice ideas! 

So far I think I'm in favor of a logo that doesn't represent the `P` of Perses.

**Comment by foxbat07** [comment]
![image (4)](https://user-images.githubusercontent.com/5414494/218590745-d6eec570-552b-4d0c-8629-b576199e1527.png)

More logo ideas done by a fellow Chronosphere designer. Have a look and tell us your favourite!

**Reply by sjcobb** [reply]
🚀 I’m in favor of the last logo, out of all the options it most associates Perses with data visualization / dashboards and could even make sense out of context if the “Perses” text wasn’t beside it

**Reply by juliusv** [reply]
Yup, same!

**Reply by Nexucis** [reply]
I'm wondering if the last one is not too close from the [Prettier's](https://prettier.io/) Logo

**Reply by AntoineThebaud** [reply]
I like the idea of a P-shaped graph like the last one, bar charts look fine and it could work with a line chart too, quick sketches I came up with:
<img src="https://user-images.githubusercontent.com/7058693/218719049-c97afe49-f00e-436d-a5be-12401fb22a91.png" width="400">

Bonus on this one, greekish-style letters that reminds the origin of the Perses name - if ever we care about this 😄. Too hard to read maybe though? Also the P here is probably too long to fit nicely into a round/square logo.
<img src="https://user-images.githubusercontent.com/7058693/218718996-edc0878a-f0e5-40b5-9fad-a6aa4d360aff.png" width="400">

**Reply by juliusv** [reply]
Hmm, I think that's a neat idea, but looks a bit too clunky in the end IMO. I wouldn't be concerned too much about the earlier P logo looking too much like another one – so many logos look almost the same these days, and it's fine :sweat_smile:

**Reply by saminzadeh** [reply]
+1 for the last one

**Reply by cndonovan** [reply]
My favorite is number 4, and then my next favorite is number 3.

**Reply by cndonovan** [reply]
I love the way number 4 suggests data/visualizations while being a "P"

**Reply by PeperoJo** [reply]
Hello everyone, I'm the designer from Chronosphere who came up with the designs for the logos - would love any feedbacks to improve it!

**Reply by Nexucis** [reply]
Thanks for your work @PeperoJo !
It looks like we have a concensus on the number 4 :).

The last question remaining is : does the color is the correct one ? Probably a question for @foxbat07 and you @PeperoJo.
We were wondering if it will fit with the blue top bar.

**Reply by PeperoJo** [reply]
Ahh I will double check the color and see if it will fit with the blue top bar.. for now, here is an animation I made for the logo

https://user-images.githubusercontent.com/8660479/220458456-5768f7d6-fd48-4a18-b42d-23326839bb5f.mov

**Reply by juliusv** [reply]
Oh that is so cool, haha! :)

**Reply by Nexucis** [reply]
ooooh wonderful !
Do you foresee it might replace the spinner used in the dashboard @PeperoJo ?
